iocore/net/P_SSLUtils.h

```diff
@@ -215,6 +217,25 @@ namespace detail
 /* namespace ssl */ } /* namespace detail */
 }
 
+struct ats_wildcard_matcher {
```

Comment:
Why were these moved up to the header from the source file? To make them inlined?